geemap/core.py

```diff
@@ -15,6 +15,8 @@
 from . import map_widgets
 from . import toolbar
 
+DRAWN_FEATURES_LAYER = "Drawn Features"
```

Comment:
How about `_DRAWN_FEATURES_LAYER` to mark this as a private variable?

Reply:
Done.